e2e/testvalues/values.go

```diff
@@ -108,3 +108,8 @@ var LocalhostClientFeatureReleases = semverutil.FeatureReleases{
 		"v7.1",
 	},
 }
+
+// HumanReadableDenomFeatureReleases represents the releases the human readable denom feature was released in.
+var HumanReadableDenomFeatureReleases = semverutil.FeatureReleases{
```

Comment:
could also call this `DenomMetadataFeatureReleases` might be the tiniest bit less wordy, but whatever, happy with this too!

Reply:
I'll maybe do the rename in #4660 to avoid needing to do a full re-run of the upgrades here!

Reply:
nvm the upgrade test failed anyway :D